### PR TITLE
feat: [IOPAE-312] add on service lifecycle change handler

### PR DESCRIPTION
**Pull request by rodoviti**
add a handler to create all "actions" to be applied to each "changed" item.

#### List of Changes

- implement a new common type RequestReviewItem into packeges/model
- implement "single item" handler
- implement unit tests

#### Motivation and Context

This change is required in order to manage (actually, to properly dispatch) every ServiceLifecycle changes

#### How Has This Been Tested?

unit tests

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/env.example [hunk 1]
```diff
@@ -32,4 +32,14 @@ AZURE_APIM=
 AZURE_APIM_RESOURCE_GROUP= 
 AZURE_SUBSCRIPTION_ID=
 
-AzureWebJobsStorage=DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://fnstorage:10000/devstoreaccount1;QueueEndpoint=http://fnstorage:10001/devstoreaccount1;TableEndpoint=http://fnstorage:10002/devstoreaccount1;
```
Comment (giuseppedipinto) [inline apps/io-services-cms-webapp/env.example:35]: Why this parameter has been removed?

Comment (rodoviti) [reply apps/io-services-cms-webapp/env.example:35]: There is no reference in either the `config.ts` or the `webapp.tf` file, so I assumed it was an unused value.
I think it is a configuration for the old version of the _tf_ module... @balanza can you confirm that?

**Comment by balanza** [comment]
(sorry, I closed the PR by mistake)

File: apps/io-services-cms-webapp/env.example (hunk 1 above)
Comment (balanza) [reply apps/io-services-cms-webapp/env.example:35]: It's a mandatory value as it's expected by Azure Function ([docs](https://learn.microsoft.com/en-my/azure/azure-functions/functions-app-settings)).

We can decide to enforce validation by adding it to `config.ts`. We don't always do it as we take the value for granted. I think the app won't even start without it (I don't remember right now)

**Review by giuseppedipinto — CHANGES_REQUESTED** [review]
Please add `AzureWebJobsStorage` according to @balanza explanation

File: apps/io-services-cms-webapp/env.example (hunk 1 above)
Comment (rodoviti) [reply apps/io-services-cms-webapp/env.example:35]: OK...so, the fact that it is not in the `webapp.tf` file (as an env var) is an error, right?

**Review by giuseppedipinto — APPROVED** [review]

**Pull request merged**